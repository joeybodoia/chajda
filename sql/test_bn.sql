CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"');
