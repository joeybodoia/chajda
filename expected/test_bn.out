CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                  spacy_lemmatize                                  
-----------------------------------------------------------------------------------
   উদাহরণ:4 পরীক্ষার:5 বাক্য:6 পরীক্ষার:8 কেসগুলি:9 উত্পন্ন:10 গুগল:14 অনুবাদে:15 রাখছি:16
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                       spacy_lemmatize                       
-------------------------------------------------------------
   উদাহরণ পরীক্ষার বাক্য পরীক্ষার কেসগুলি উত্পন্ন গুগল অনুবাদে রাখছি
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                      spacy_lemmatize                                                      
---------------------------------------------------------------------------------------------------------------------------
   এটি:2 আমার:3 উদাহরণ:4 পরীক্ষার:5 বাক্য:6 যা:7 পরীক্ষার:8 কেসগুলি:9 উত্পন্ন:10 করার:11 জন্য:12 আমি:13 গুগল:14 অনুবাদে:15 রাখছি:16
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                   spacy_lemmatize                                    
--------------------------------------------------------------------------------------
   এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                         spacy_lemmatize                                          
--------------------------------------------------------------------------------------------------
   ":2 উদাহরণ:5 পরীক্ষার:6 বাক্য:7 পরীক্ষার:9 কেসগুলি:10 উত্পন্ন:11 গুগল:15 অনুবাদে:16 রাখছি:17 ।:18 ":19
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                          spacy_lemmatize                          
-------------------------------------------------------------------
   " উদাহরণ পরীক্ষার বাক্য পরীক্ষার কেসগুলি উত্পন্ন গুগল অনুবাদে রাখছি । "
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                             spacy_lemmatize                                                              
------------------------------------------------------------------------------------------------------------------------------------------
   ":2 এটি:3 আমার:4 উদাহরণ:5 পরীক্ষার:6 বাক্য:7 যা:8 পরীক্ষার:9 কেসগুলি:10 উত্পন্ন:11 করার:12 জন্য:13 আমি:14 গুগল:15 অনুবাদে:16 রাখছি:17 ।:18 ":19
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                      spacy_lemmatize                                       
--------------------------------------------------------------------------------------------
   " এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি । "
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                  spacy_lemmatize                                  
-----------------------------------------------------------------------------------
   উদাহরণ:4 পরীক্ষার:5 বাক্য:6 পরীক্ষার:8 কেসগুলি:9 উত্পন্ন:10 গুগল:14 অনুবাদে:15 রাখছি:16
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                       spacy_lemmatize                       
-------------------------------------------------------------
   উদাহরণ পরীক্ষার বাক্য পরীক্ষার কেসগুলি উত্পন্ন গুগল অনুবাদে রাখছি
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                      spacy_lemmatize                                                      
---------------------------------------------------------------------------------------------------------------------------
   এটি:2 আমার:3 উদাহরণ:4 পরীক্ষার:5 বাক্য:6 যা:7 পরীক্ষার:8 কেসগুলি:9 উত্পন্ন:10 করার:11 জন্য:12 আমি:13 গুগল:14 অনুবাদে:15 রাখছি:16
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                   spacy_lemmatize                                    
--------------------------------------------------------------------------------------
   এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                         spacy_lemmatize                                          
--------------------------------------------------------------------------------------------------
   ":2 উদাহরণ:5 পরীক্ষার:6 বাক্য:7 পরীক্ষার:9 কেসগুলি:10 উত্পন্ন:11 গুগল:15 অনুবাদে:16 রাখছি:17 ।:18 ":19
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                          spacy_lemmatize                          
-------------------------------------------------------------------
   " উদাহরণ পরীক্ষার বাক্য পরীক্ষার কেসগুলি উত্পন্ন গুগল অনুবাদে রাখছি । "
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                             spacy_lemmatize                                                              
------------------------------------------------------------------------------------------------------------------------------------------
   ":2 এটি:3 আমার:4 উদাহরণ:5 পরীক্ষার:6 বাক্য:7 যা:8 পরীক্ষার:9 কেসগুলি:10 উত্পন্ন:11 করার:12 জন্য:13 আমি:14 গুগল:15 অনুবাদে:16 রাখছি:17 ।:18 ":19
(1 row)

SELECT spacy_lemmatize('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                      spacy_lemmatize                                       
--------------------------------------------------------------------------------------------
   " এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি । "
(1 row)

SELECT spacy_tsvector('bn',' "এটি আমার উদাহরণ পরীক্ষার বাক্য যা পরীক্ষার কেসগুলি উত্পন্ন করার জন্য আমি গুগল অনুবাদে রাখছি।"');
                                     spacy_tsvector                                      
-----------------------------------------------------------------------------------------
 'অনুবাদে':15 'উত্পন্ন':10 'উদাহরণ':4 'কেসগুলি':9 'গুগল':14 'পরীক্ষার':5,8 'বাক্য':6 'রাখছি':16
(1 row)

